src/client/components/app-core/layout.css

```diff
@@ -8,7 +8,7 @@ html {
 }
 
 body {
-  overflow: hidden;
+  overflow: scroll;
```

Comment:
It's a shame that we now force the scroll-bar, all the time. Is this needed to print the PDF document properly? Could it be toggled strictly during the printing process?

Reply:
Ow crap, that was a debug feature which needs to be removed